unfoldingWord/translationCore
# Added Support For tStudio Single Chunks

#### This pull request addresses:
This PR handles what happens when an import source contains no verse marker.


#### How to test this pull request:
Ensure tests are passing
Try loading the project described in #2642 , and ensure that checks 3:3, and 3:8 are present.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3128)
